### PR TITLE
Update for Verify APIs and removing Company and EmailVerification support

### PR DESCRIPTION
**Pull request by AAnandFC**
Story: sc-143553

### CODE REVIEW

**Comment by AAnandFC** [comment]
This has breaking changes and hence will be published as v2.0.0

File: README.md [hunk 1]
```diff
@@ -45,7 +45,6 @@ your integration.
 - _[Enrich](https://platform.fullcontact.com/docs/apis/enrich/introduction)_
```
Comment (prakhar-j) [inline README.md:45]: Can you update these links, all are now under docs.fullcontact.com

File: README.md [hunk 2]
```diff
@@ -615,3 +571,184 @@ if resp.IsSuccessful {
...
+- `verify.signals`
+
+### Verify Request
+Verify accepts a `MultifieldRequest` as its input for
+
+- `verify.activity`
+- `verify.match`
+- `verify.signals`
+
+All Verify APIs requires `MultifieldRequest` requests, which can be constructed by using `NewMultifieldRequest` and following are it's parameters.
```
Comment (prakhar-j) [inline README.md:588]: Lets create a section for MultiFieldRequest, and refer it here (and in Person and Resolve requests), so there is less duplicate. You can refer Java client lib Readme.

File: fc/fullcontact.go [hunk 3]
```diff
@@ -8,6 +8,30 @@ import (
 	"time"
 )
 
+func (fcClient *fullContactClient) newHttpRequest1(url string, reqBytes []byte) (*http.Request, error) {
```
Comment (prakhar-j) [inline fc/fullcontact.go:11]: If you have combined both request types here, then we can remove individual methods below?

File: fc/fullcontact.go [hunk 4]
```diff
@@ -542,6 +529,84 @@ func (fcClient *fullContactClient) PermissionVerify(permissionRequest *Permissio
...
+	ch, reqBytes, err := doVerfiyAPIPrerequisites(multifieldRequest)
+	if err != nil {
+		go sendToChannel(ch, nil, "", err)
+		return ch
+	}
+	go fcClient.do(verifyActivityUrl, reqBytes, ch)
+	return ch
+}
+
+func doVerfiyAPIPrerequisites(multifieldRequest *MultifieldRequest) (chan *APIResponse, []byte, error) {
```
Comment (prakhar-j) [inline fc/fullcontact.go:597]: We can make this method maybe "validate and marshall multifieldreq" and use it in permission too?

**Review by prakhar-j — CHANGES_REQUESTED** [review]
Minor comments, looks good. Thanks!

File: README.md (hunk 1 above)
Comment (AAnandFC) [reply README.md]: Updated for Enrich, Permission and Resolve

File: README.md (hunk 2 above)
Comment (AAnandFC) [reply README.md:588]: @prakhar-j  Is this the java lib? https://github.com/fullcontact/fullcontact-java-client

Didn't see any section of MFR in this repo

File: fc/fullcontact.go (hunk 3 above)
Comment (AAnandFC) [reply fc/fullcontact.go:11]: I created this method, but have't used it. Forgot about adding this as well :)

File: fc/fullcontact.go (hunk 4 above)
Comment (AAnandFC) [reply fc/fullcontact.go]: Made the change. In addition to it, moved the request call also to the new method

File: README.md (hunk 2 above)
Comment (prakhar-j) [reply README.md:588]: Should be https://github.com/fullcontact/fullcontact-java-client/tree/master/java11#multifieldrequest and https://github.com/fullcontact/fullcontact-java-client/tree/master/java11#building-a-person-enrichresolve-request

Comment (AAnandFC) [reply README.md]: go client readme updated to mirror the docs for java client

**Review by prakhar-j — CHANGES_REQUESTED** [review]

**Review by jidler — APPROVED** [review]

**Review by prakhar-j — APPROVED** [review]

**Pull request merged**